src/gui/guiFormSpecMenu.cpp

```diff
@@ -1412,15 +1575,22 @@ void GUIFormSpecMenu::parseTabHeader(parserData* data, const std::string &elemen
 		spec.ftype = f_TabHeader;
 
 		v2s32 pos;
-		{
+		v2s32 geom;
+
+		if (data->real_coordinates) {
+			pos = getRealCoordinateBasePos(false, &v_pos);
+			std::vector<std::string> v_geom = {"1", h_geom}; // Dummy value
+			geom = getRealCoordinateGeometry(&v_geom);
+			pos.Y -= geom.Y; // TabHeader base pos is the bottom, not the top.
+		} else {
 			v2f32 pos_f = pos_offset * spacing;
 			pos_f.X += stof(v_pos[0]) * spacing.X;
 			pos_f.Y += stof(v_pos[1]) * spacing.Y - m_btn_height * 2;
 			pos = v2s32(pos_f.X, pos_f.Y);
+
+			geom.Y = m_btn_height*2;
```

Comment:
```suggestion
			geom.Y = m_btn_height * 2;
```
In this code block, spaces are used.

Reply:
There's a lot of things like this, and I don't want to fix them all because that would bloat the PR and make it harder to review.  I fix some of them, though.